docs/how-to-guides/contributing/documentation.rst

```diff
@@ -44,4 +53,4 @@ Using Nix
 
 We provide a ``nix`` configuration that provides all the required libraries for developing the
 renku helm charts and the documentation. To use, simply run ``nix-shell`` and your shell will
-be configured with everything you need.
+be configured with everything you need.
```

Comment:
Nitpicking : could you please preserve the newline at the end of the file? 😇

Reply:
Done!